### PR TITLE
javascript-if - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-07-07 at 3 59 03 PM
Screen Shot 2022-07-07 at 4 00 14 PM
Screen Shot 2022-07-07 at 4 03 04 PM

### CODE REVIEW

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-07 at 4 38 51 PM

**Comment by yuliia-p** [comment]
Screen Shot 2022-07-07 at 5 09 13 PM

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 16) {
+    return false;
+  } else if (person.age >= 21) {
+    return false;
+  }
+}
+
+var msg = ' ';
```
Comment (codingcodymiller) [inline javascript-if/main.js:39]: You should have this `msg` variable inside the `categorizeAcidity` function

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    msg = 'acid';
+  } else if ((pH <= 14) && (pH > 7)) {
+    msg = 'base';
+  } else {
+    msg = 'invalid pH level';
+  }
+  return msg;
+}
+
+var catchPhrase = ' ';
```
Comment (codingcodymiller) [inline javascript-if/main.js:54]: Same here, this should be within the function definition

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**